runcoms/README.md

```diff
@@ -10,61 +10,199 @@ in */etc* by default.
+
+This can be solved by calling (or aliasing) your terminal emulator with 
+the required option to invoke a *login* shell, for `XTerm`, for example 
+you would call `xterm -e zsh -ls` on tmux you may add one of the following 
+lines to your `.tmux.conf`:
+
+```config
+#tmux.conf:
+
+        set -g default-command 'exec /usr/bin/zsh -ls' #this will make zsh a login shell AND a session leader
+        set -g default-shell '/usr/bin/zsh -ls'
+```
+
+Here is an examplefor *i3wm* on ubuntu
+```bash
+
+#i3wm.conf:
+        ...
+        set $XTERM_CMD='xterm -e zsh -ls`
+        bind $mod+x $XTERM_CMD
+        ...
+```
+if you want to replace your non-login shell on your emulator with a brand-new 
+shiny *login* shell you can issue `exec zsh -ls` at your command prompt. 
+Doing this, however **will not write that terminal’s history to file** unless
+`INC_APPEND_HISTORY` was set when you strted the shell 
+
+`INC_APPEND_HISTORY` makes `zsh` append history entries as soon as the 
+command is finished. *zprezto* helpfuly sets `INC_APPEND_HISTORY`, 
+`INC_APPEND_HISTORY_TIME`, and `SHARE_HISTORY` which makes your history 
+available immediately from different terminals on command completion, 
+and records execution time on the history file. Pretty neat, huh?)
+
+You could tack-on all interactive customizations on `zshrc` but you will
+end up with bloated shells on non-interactive environments. Perhaps desktop 
+workstation users will not notice any difference, but in resource-constrained 
+environments it may be significant in terms of execution time and energy 
+consumption (v.gr when running a Raspberry PI on batteries or on a cell 
+phone environment such as `termux`)
+
+You may be thinking... why is `zpreztorc` not invoked on `.zlogin` instead 
+of `.zshrc`? I believe it was a wise design decision, in order to 
+eliminate complexity by having the whole of *prezto* configured and 
+launched from a single point, as some of the modules need to be sourced 
+early in the startup process, `editor` and `gnu-utls` come to mind. 
+If invoked later on the startup, such as would be the case in `.zlogin` 
+would makee it easier to conflict with stuff users may add on `zshrc`. 
+Also, it avoids **not being loaded** and causing user frustration if 
+the user launches *non-login shells* from graphical environments, as 
+discussed above.
+
+## Conclusion
+
+For customization of your environment, use your `.zshrc` after zprezto 
+has been invoked, or before to remove or activate packages via `zstyle` 
+calls 
+ 
+ or
+
+use `.zshlogin` to override or tweak zprezto behavior. there is also
+`.zpreztorc` but you may want to leave that one alone.
+
+In any case, this question is often brought up by users that use the 
+distributed *runcoms* **as is** which is a terrible idea, since doing
+`git pull` to update your distribution will clobber your customizations.
+
+it’s a better idea to *copy* the distributed *runcoms* to your `$ZDOTDIR` a
```

Comment:
This is a bit of a larger discussion. There have been talks about adding `zlogin.local` (and equivalents for the other files) which would be automatically sourced in the official prezto runcoms... and there's been talk of copying rather than symlinking. There hasn't been a general consensus yet, but that's mostly because noone has really taken the initiative to push a change like that through. I would be fine with either solution.

The original workflow with prezto was essentially fork the repo and make your own tweaks and changes there... which I understand, but I think it does the opposite of what was intended - a smaller amount of useful changes make it back into the main repo. I believe *this* is the reason the runcoms are symlinked. It also removes one step of debugging if you're using the official runcoms... because we don't need to go through all the zsh startup files and figure out where loading prezto went wrong.

I don't know what to do here, but I'd be happy to see a larger discussion come from this (Probably in a new issue)... however, even if I don't agree with it, I don't think documentation included with the prezto repo should be saying that the official recommendation (at least what we're *currently* recommending) is a "terrible idea" and that it's a better idea to do something different from what we suggest in the main README.

That being said, there are a number of easy to make mistakes, so providing a list of potential pitfalls and how to avoid them would be useful.

Reply:
Thank you for your feedback! I'll try and incorporate your observations, and yup there are a bunch of typos and dyslexyc mistakes. let's see how the second round comes up. =)